commit dc366364c4ef809dccd063919314301f8ba01ac2
Author: Thomas Hellstrom <thellstrom@vmware.com>
Date:   Thu Mar 22 10:15:23 2018 +0100

    drm/vmwgfx: Fix multiple command buffer context use
    
    The start / stop and preempt commands don't honor the context argument
    but rather acts on all available contexts.
    
    Also add detection for context 1 availability.
    
    Note that currently there's no driver interface for submitting buffers
    using the high-priority command queue (context 1).
    
    Testing done:
    Change the default context for command submission to 1 instead of 0,
    verify basic desktop functionality including faulty command injection and
    recovery.
    
    Signed-off-by: Thomas Hellstrom <thellstrom@vmware.com>
    Reviewed-by: Brian Paul <brianp@vmware.com>
    Reviewed-by: Deepak Rawat <drawat@vmware.com>

diff --git a/drivers/gpu/drm/vmwgfx/vmwgfx_drv.c b/drivers/gpu/drm/vmwgfx/vmwgfx_drv.c
index 184340d486c3..5055e5f68c4f 100644
--- a/drivers/gpu/drm/vmwgfx/vmwgfx_drv.c
+++ b/drivers/gpu/drm/vmwgfx/vmwgfx_drv.c
@@ -301,6 +301,8 @@ static void vmw_print_capabilities(uint32_t capabilities)
 		DRM_INFO("  Guest Backed Resources.\n");
 	if (capabilities & SVGA_CAP_DX)
 		DRM_INFO("  DX Features.\n");
+	if (capabilities & SVGA_CAP_HP_CMD_QUEUE)
+		DRM_INFO("  HP Command Queue.\n");
 }
 
 /**